### ISSUE TITLE
Documentation: Add FAQ page demonstrating how to create a WinForms control that is configurable in the designer

### ISSUE THREAD

**Issue by wdc63** [ScottPlot/ScottPlot#2167]
I want users to choose variables to create charts at runtime. Do you have a related demo?

**Comment by swharden** [comment]
Hi @wdc63, 

I do not have any examples currently, but a simple code example may be a useful thing to features on a new FAQ page! If you make progress toward implementing this, I'd be appreciative if you could share the code you used here and I can add it to the page.

I'll update this issue to track progress toward creating such an FAQ page. If you or someone else gets to it before I do (probably not for a few days), I'd be happy to accept a PR and merge it in quickly (#2010). Note that FAQ pages can be created just by adding a markdown folder here: https://github.com/ScottPlot/ScottPlot.NET/tree/main/content/faq

Thanks for your question!

**Comment by wdc63** [comment]
Thank you Scott, I just discovered ScottPlot today, and before that I researched a bunch of commercial .net charting libraries such as devexpress, scichart, etc. which are fully functional but too bulky and expensive. ScottPlot has done a very good job in terms of function, performance and visual effects. I will study ScottPlot carefully in the next period of time. If there is any progresses, I will go to the FAQ page to write them.
